compendium/modules/w04-objects-exercise.tex

```diff
@@ -332,7 +319,7 @@
 
 \begin{REPL}
 > wget -O introprog_2.13-1.1.4.jar http://cs.lth.se/pgk/lib
```

Comment:
There is new jar for Scala 3 introprog_3-1.1.5.jar, but that needs to be uploaded to http://cs.lth.se/pgk/lib 
Issue here: https://github.com/lunduniversity/introprog-scalalib/issues/22